spec/rspec/filtering_spec.rb

```diff
@@ -16,6 +16,16 @@
       ).as_null_object
     end
 
+    context 'when filter manager has no inclusions defined' do
+      let(:files) { %w[./spec/foo_spec.rb] }
+      let(:inclusions) { {} }
+
+      it 'does not change any filtering rules' do
+        remove_unnecessary_filters
+        expect(inclusions[:ids]).to be_nil
```

Comment:
If you want to test that `remove_unnecessary_filters` do not modify `inclusions[:ids]` you can use more descriptive syntax
`expect { remove_unnecessary_filters }.not_to change { inclusions[:ids] }`